non_linear/graphs/floyd_warshall/floyd_warshall.py

```diff
@@ -0,0 +1,40 @@
+def adjacencyListToMatrix(adjList, vertices):
+    matrix = [[float('inf')] * vertices for _ in range(vertices)]
+    for i in range(vertices):
+        matrix[i][i] = 0  # Distance to self is 0
+    for u in adjList:
+        for v, weight in adjList[u]:
+            matrix[u][v] = weight
```

Comment:
Maybe a more pythonic style?

```suggestion
    for u, edges in adjList.items():
        for v, weight in edges:
            matrix[u][v] = weight
```